diagnostics/app/model/diagnostics/css/Css.scala

```diff
@@ -0,0 +1,33 @@
+package model.diagnostics.css
+
+import play.api.libs.json._
+import collection.Map
+
+case class CssReport(
+  href: String,
+  breakpoint: String,
+  className: String,
+  contentType: String,
+  selectors: JsObject ) {
+
+  override def toString(): String = {
+    val selectorMap = selectors.value.asInstanceOf[Map[String, JsBoolean]]
+    selectorMap.map {
+      case (key: String, exists:JsBoolean) => s"css ; $key ; $exists ; $contentType ; $breakpoint ; $href"
+    }.mkString("\n")
+  }
+}
+
+object CssReport {
+  implicit val reads = Json.reads[CssReport]
+}
+
+object Css extends common.Logging {
+
+  def report(requestBody: JsValue): Unit = {
+    requestBody.validate[CssReport] match {
+      case JsSuccess(report, _) => log.info("\n" + report.toString)
```

Comment:
I think they block, yeah. Akka has a thing that does asynchronous logging. Though I kinda think it's a bit of a premature optimisation unless you've seen it's causing a problem

CC @philwills Am I right in this?